docs/src/fibers.md

```diff
@@ -159,7 +196,33 @@ SparseCOO (0.0) [1:4,1:3]
 
 The COO format is compact and straightforward, but doesn't support random
 access. For random access, one should use the `SparseHash` format. A full listing
-of supported formats is described below:
+of supported formats is described after a rough description of shared common internals of level,
+relating to types and storage.
+
+## Types and Storage of Level
+
+All levels have an `indextype` and a `postype`, typically denoted as `Ti` and `Tp` in the constructors. 
+Index type is mostly self expanatory as it is ofen just `Int`, but for levels that store multiple dimensions, 
+the index type is often a subtype of a Tuple. Pos (or position) type is the type used to define the iteratiorator through any 
```

Comment:
spelling

Reply:
Also, I think the description of what `indextype` is a little unclear here. Can we narrow the scope of the indextype trait to be:

```
"""
    indextype(lvl)
Return an index type with the same flavor as those used to index the fibers contained in `lvl`.
"""
function indextype end
```